CoffeeChat.xcodeproj/project.pbxproj

```diff
@@ -1038,6 +1042,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = ZGMCXU7X3U;
 				INFOPLIST_FILE = CoffeeChat/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
not sure about stats on ios version adoption, but we might wanna bump this down further. i know I'm definitely not running ios 14 on my phone